app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -100,6 +101,19 @@ ShortFormApplicationService = ($localStorage) ->
   Service.cancelHouseholdMember = ->
     householdMembers = Service.householdMembers.filter (m) ->
       (m != Service.householdMember && m.id != Service.householdMember.id)
+    # clear out preferences where this householdMember was selected
```

Comment:
@chrisdolendo yeah I totally agree, nested would be better, doing the string check `indexOf('_household_member')` to see if it's a "household member"-type of key is a little bit hacky vs. just marking explicitly that it is `{ preferenceName: { householdMember: "Jane" }}`. I also agree it's not a big deal for what we have. 

You'll also see I slightly refactored the preferences to move them into the `application` namespace instead of `applicant` since they apply to the whole application/household (and not the applicant's own personal data like address, etc.) So you may want to merge this in with the work you're currently doing on #120716389 (or else we can just merge after that story is done).